Software/facial_recognition/capture.py

```diff
@@ -3,24 +3,42 @@
 # Based on code by Marcelo Rovai -> https://github.com/Mjrovai/OpenCV-Face-Recognition
 # Program to capture face snapshots
 
-import cv2, argparse
+import cv2, argparse, json, os
 
 def arg_parser():
     parser = argparse.ArgumentParser()
 
     parser.add_argument("-c", "--classifier", default="haarcascade_frontalface_default.xml", help="Path to the XML Cascade Classifier file")
-    parser.add_argument("-i", "--id", type=int, help="User ID that will correspond to pictures taken")
+    parser.add_argument("-u", "--user", help="Username corresponding to the pictures taken: ")
     parser.add_argument("-n", "--num", type=int, default=100, help="Number of photos to capture")
 
     args = parser.parse_args()
 
-    if args.id is None:
-        args.id = input("Please enter the user ID: ")
+    if not args.user:
+        args.user = input("Please enter the user name: ")
 
     return args
 
+def save_user(username):
+    file = "users.txt"
+    users = []
+
+    if os.path.exists(file):
+        with open(file, "r") as f:
+            for line in f.readlines():
+                users.append(line.strip())
+
+    if username in users:
+        for id, name in enumerate(users):
+            if username == name:
+                return id
```

Comment:
I think you can use .indexOf() here, would be slightly cleaner

Reply:
Done